src/plugins/ui/evaluation_conditional_format.ts

```diff
@@ -459,6 +459,10 @@ export class EvaluationConditionalFormatPlugin extends UIPlugin {
 
     currentRanges = currentRanges.concat(toAdd);
     const newRange: string[] = recomputeZones(currentRanges, toRemove);
+    if (newRange.length === 0) {
+      this.dispatch("REMOVE_CONDITIONAL_FORMAT", { id: cf.id, sheetId });
+      return;
+    }
```

Comment:
Instead of adding the "REMOVE" command before the "ADD" command, can't we simply make the ADD command correctly remove the previous range ?

Reply:
The point here is to remove the CF if it has no ranges (we don't dispatch ADD command afterward). 

The previous implementation dispatched a ADD_CF with empty ranges, which was rejected by the allowDispatch. Which is fine, because we don't wan't to be allowed to create a new CF with no ranges.

Reply:
My bad, I missed the "return" after the remove. Ok for me then :)